meetuppizza/forms.py

```diff
@@ -0,0 +1,10 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+
+
+class RegistrationForm(UserCreationForm):
```

Comment:
This form is totally testable, but I don't see any unit tests for it. :unamused:

How did it spring into existence without failing unit tests?